dashboard/app/main.html

```diff
@@ -14,7 +14,10 @@
 <body>
 	{{template "header" .Header}}
 	{{if $.Decommissioned}}<h1>This kernel is DECOMMISSIONED</h1>{{end}}
-	{{template "manager_list" $.Managers}}
+        {{if not $.BugFilter.Filter.Any}}
+		{{template "manager_list" $.Managers}}
```

Comment:
We used to show the single manager for ?manager=/only_manager= requests.
That was handy to show/monitor state of a single instance, e.g. just wanted to give https://syzkaller.appspot.com/upstream?only_manager=ci-qemu2-riscv64 link to riscv maintainers.
With this change it does not work anymore. It was useful, I think we need to restore it.